Justfile

```diff
@@ -34,11 +34,11 @@ cargo_test_invocation := "PKGDB_BIN=${PKGDB_BIN} cargo test --workspace"
 
 # Build the compilation database
 build-cdb:
-    @make -C pkgdb -j -s cdb
+    @make -C pkgdb -j 4 -s cdb
```

Comment:
How is this related to changing the version of nixpkgs we're tracking?

Reply:
It isn't, it crept in as a fix to avoid OOM-ing. removing....